ZenPacks/zenoss/Microsoft/Windows/datasources/IISSiteDataSource.py

```diff
@@ -162,7 +162,16 @@ def collect(self, config):
         if not iis_version:
             winrs = IISCommander(conn_info)
             version = yield winrs.get_iis_version()
-            iis_version = version.stdout[0]
+            # version should be in 'Version x.x' format
+            # 7 and above use the same namespace/query
+            try:
+                iis_version = re.match('Version (\d).*', version.stdout[0]).group(1)
```

Comment:
This seems like it could also raise an AttributeError if `re.match()` doesn't match because you'll be attempting to call `None.group(1)`.